types/graphql-upload/index.d.ts

```diff
@@ -1,53 +1,27 @@
-// Type definitions for graphql-upload 8.0
+// Type definitions for graphql-upload 15.0
 // Project: https://github.com/jaydenseric/graphql-upload#readme
 // Definitions by: Mike Marcacci <https://github.com/mike-marcacci>
+//                 Simon Feiden <https://github.com/rdsfj>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 4.1
+// TypeScript Version: 4.8
 
-/* tslint:disable:no-unnecessary-generics */
+import { ReadStream } from 'fs-capacitor';
```

Comment:
This needs to be empty except for metadata since the library doesnt have index.js so having types in index.d.ts doesnt represent correct structure of package.

Reply:
I did not manage to leave this file completely empty because of the `no-useless-files` linting rule.
If I disable the rule at the very top of the file, the meta data parser breaks because it expects `// Type definitions` in the first line. Disabling the rule later in the file will not satisfy tslint either, it will report the error anyway.
I found no other package with an empty `index.d.ts` and I also didn't want to disable the rule in `tslint.json`.
To solve this, I added a single definition to make tslint happy. Not super nice, so let me know if there is a better way to solve this.